mlflow/tracking/_model_registry/fluent.py

```diff
@@ -662,9 +660,21 @@ def load_prompt(
 
     """
     client = MlflowClient()
-    prompt = client.load_prompt(
-        name_or_uri=name_or_uri, version=version, allow_missing=allow_missing
-    )
+
+    # Handle URI vs name+version cases
+    if name_or_uri.startswith("prompts:/"):
+        # For URIs, don't pass version parameter
+        prompt = client.load_prompt(name_or_uri, allow_missing=allow_missing)
+    else:
+        # For names, version is required
+        if version is None:
+            raise MlflowException(
+                "Version must be specified when loading a prompt by name. "
+                "Use a prompt URI (e.g., 'prompts:/name/version') or provide the version "
+                "parameter.",
+                INVALID_PARAMETER_VALUE,
+            )
+        prompt = client.load_prompt(name_or_uri, version=version, allow_missing=allow_missing)
```

Comment:
Seems like we're repeating / duplicating this URI check logic from `mlflow/tracking/client.py`. Can we deduplicate, e.g. by adding a utility?